source/JsMaterialX/test/Element.spec.js

```diff
@@ -0,0 +1,145 @@
+import { expect } from 'chai';
+import Module from './_build/JsMaterialX.js';
+
+describe('Element', () => {
+  let mx, doc, valueTypes;
+
+  const primitiveValueTypes = {
+    Integer: 10,
+    Boolean: true,
+    String: 'test',
+    Float: 15,
+    IntegerArray: [1,2,3,4,5],
+    FloatArray: [12, 14], // Not using actual floats to avoid precision problems
+    StringArray: ['first', 'second'],
+    BooleanArray:[true, true, false],
+  }
+
+  before(async () => {
+      mx = await Module();
+      doc = mx.createDocument();
+      valueTypes = {
```

Comment:
`valueTypes` doesn't seem to be modified. Could it also be defined as a global const, like `primitiveValueTypes`?

Reply:
it depends on mx being initialised. That is why it is in the before :(